script/test

```diff
@@ -152,7 +152,7 @@ do_check()
 {
     (cd "${OTBR_TOP_BUILDDIR}" \
         && ninja && sudo ninja install \
-        && CTEST_OUTPUT_ON_FAILURE=1 ninja test)
+        && CTEST_OUTPUT_ON_FAILURE=1 sudo ninja test)
```

Comment:
only run `otbr-posix-gtest-unit` with sudo?

Reply:
Updated to use 'ctest' and add a label 'sudo' for tests that need to be run with sudo. With '-L' and '-LE' we can run two sets of tests.